packages/dashboard-frontend/src/typings/che.d.ts

```diff
@@ -11,23 +11,6 @@
  */
 
 declare namespace che {
-  export interface Workspace {
-    id: string;
-    projects?: any;
-    links?: {
-      ide?: string;
-      [rel: string]: string | undefined;
-    };
-    temporary?: boolean;
-    status: string;
-    namespace?: string;
-    attributes?: WorkspaceAttributes;
-    devfile: WorkspaceDevfile;
-    runtime?: WorkspaceRuntime;
-    isLocked?: boolean;
-    usedResources?: string;
-  }
-
```

Comment:
Please remove all interfaces related to Che7 workspaces.

Reply:
I have fixed it.